### PR TITLE
Add strings for video mode

### PR DESCRIPTION
**Pull request by MortimerGoro**
Add these before waiting for the video mode landing so they can be translated earlier

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -166,6 +166,8 @@
     <string name="cancel_button">Cancel</string>
     <!-- This string labels a button that are used to approve an action. -->
     <string name="ok_button">OK</string>
+    <!-- This string labels a button that are used to approve an action. -->
```
Comment (keianhzo) [inline app/src/main/res/values/strings.xml]: Maybe "This string labels a button to go back to the previous menu or exit the current mode"?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -181,4 +183,42 @@
...
+    <!-- This string is displayed when selecting a background brightness mode -->
+    <string name="brightness_mode_dark">Dark</string>
+    <!-- This string is displayed when selecting a background brightness mode -->
+    <string name="brightness_mode_void">Void</string>
+    <!-- This string labels a button to open the Video projection mode selector-->
+    <string name="video_mode_tooltip">Video Projection</string>
+    <!-- This string is displayed when selecting a video playback mode -->
+    <string name="video_mode_3d_side">3D Side by Side</string>
+    <!-- This string is displayed when selecting a video playback mode -->
+    <string name="video_mode_360">360</string>
```
Comment (keianhzo) [inline app/src/main/res/values/strings.xml]: Shouldn't numbers be in the non_I10n?

Comment (MortimerGoro) [reply app/src/main/res/values/strings.xml]: I wasn't sure all the languages use `180` or `360` to refer to that kind of videos.

**Comment by keianhzo** [comment]
Let's the I10n team decide about the numbers when they check.

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (cvan) [reply app/src/main/res/values/strings.xml]: numerals can be localised, so yeah we should keep. a better L10n note might say `360` refers to `360°` (or `360-degree` videos)

**Pull request merged**